spec.html

```diff
@@ -37836,6 +37836,63 @@ <h1>RegExp.prototype</h1>
         <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
       </emu-clause>
 
+      <emu-clause id="sec-regexp.escape" number="2">
+        <h1>RegExp.escape ( _S_ )</h1>
+        <p>This function returns a copy of _S_ in which characters that are potentially special in a regular expression |Pattern| have been replaced by equivalent escape sequences.</p>
+        <p>It performs the following steps when called:</p>
+
+        <emu-alg>
+          1. If _S_ is not a String, throw a TypeError exception.
+          1. Let _escaped_ be the empty String.
+          1. Let _cpList_ be StringToCodePoints(_S_).
+          1. For each code point _c_ in _cpList_, do
```

Comment:
```suggestion
          1. For each code point _c_ of _cpList_, do
```
(When `For each` iterates over the elements of a List, we say "of" there.)